app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt

```diff
@@ -316,36 +311,13 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
     /*region TEI*/
     override fun displayTeiInfo(teiInfo: TeiAttributesInfo) {
         if (mode != EnrollmentMode.NEW) {
-            binding.title.visibility = View.GONE
-            binding.teiDataHeader.root.visibility = View.VISIBLE
-
-            binding.teiDataHeader.mainAttributes.apply {
-                text = teiInfo.teiMainLabel(getString(R.string.tracked_entity_type_details))
-                setTextColor(Color.WHITE)
-            }
-            when (val secondaryLabel = teiInfo.teiSecondaryLabel()) {
-                null -> binding.teiDataHeader.secundaryAttribute.visibility = View.GONE
-                else -> {
-                    binding.teiDataHeader.secundaryAttribute.text = secondaryLabel
-                    binding.teiDataHeader.secundaryAttribute.setTextColor(Color.WHITE)
-                }
-            }
-
-            if (teiInfo.profileImage.isEmpty()) {
-                binding.teiDataHeader.teiImage.visibility = View.GONE
-                binding.teiDataHeader.imageSeparator.visibility = View.GONE
-            } else {
-                Glide.with(this).load(File(teiInfo.profileImage))
-                    .transition(DrawableTransitionOptions.withCrossFade())
-                    .transform(CircleCrop())
-                    .into(binding.teiDataHeader.teiImage)
-                binding.teiDataHeader.teiImage.setOnClickListener {
-                    presenter.onTeiImageHeaderClick()
-                }
-            }
+            binding.title.text =
+                resourceManager.formatWithEnrollmentLabel(
```

Comment:
It makes no sense to use an empty resource. Just call the defaultEnrollmentLabel method you added.